src/test/java/com/philips/research/spdxbuilder/persistence/tree/TreeFormatsTest.java

```diff
@@ -83,6 +83,22 @@ void gradle() {
                     new Package("com.group", "upgraded", "2.3.4"));
         }
 
+        @Test
+        void nuget() {
+            format.configure(parser, "nuget");
+
+            parse("ProjectName 1.0.0-dev",
+                    "---IdentityModel.AspNetCore 3.0.0.0",
+                    "AnotherProjectName 1.0.1",
```

Comment:
If "digits" are a criteria to match the version, then you need to add a sample where the name includes digits. (This ensures a part of the name does not show up in the detected version.)

Reply:
Done